src/main/java/taxi/controller/driver/CreateDriverController.java

```diff
@@ -0,0 +1,36 @@
+package taxi.controller.driver;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import taxi.lib.Injector;
+import taxi.model.Driver;
+import taxi.service.DriverService;
+
+public class CreateDriverController extends HttpServlet {

```

Comment:
думаю не стоит создавать отдельный контролер для создания водителей.
достаточно изменить AddDriverController

Reply:
I have them simply are responsible for different functions. The task says that CreateDriverController should ignore the filter, and logically, I thought that a new user registration could be hung on it. Everything seemed to be logically intertwined in this case, but it would be more logical to call it RegistrationController. One way or another in the task said "CreateDriverController should be available for non authenticated users", and at the same time there wasn't any mention about deleting AddDriverController which was at the beginning. Hope Q&A will set things right...